Documentation/Manual/Texinfo/extensions.texi

```diff
@@ -10,11 +11,22 @@
 * Raster Images::
 @end menu
 
+@node Frame and sheet icons
+@section Frame and sheet icons
+
+According to the @clim{} specification, an icon can be defined for an
+application frame class using the @initarg{:icon} initarg of the
+@fmacro{define-application-frame} macro.  @mcclim{} extends this design to
+top-level sheets and adds
+
+@include docstrings/fun-clim-extensions-frame-icon.texi
+@include docstrings/fun-clim-extensions-setf-frame-icon.texi
```

Comment:
also sheet-icon and setf-sheet-icon.texi?

Reply:
Fixed.